src/program/program.resolver.ts

```diff
@@ -265,10 +265,24 @@ export class ProgramResolver {
 	}
 
 	@Mutation("updateContent")
-	async updateContent(@Args("input") input: ContentFields) {
+	/* Defined new argument inputArray below as an array of ContentFields inorder to check 
+	whether any of the elements in content array has the primary attribute set as true */
+	async updateContent(
+		@Args("input") input: ContentFields,
+		@Args("inputArray") inputArray: ContentFields[]
+	) {
+		/* Whenever user tries to update certain content's primary attribute in the front end, 
+		that particular content's id is passed to the constant passedId defined below */
+		const passedId = "";
+		if (inputArray) {
```

Comment:
I'm unsure as to why you added this section to the resolver. We are concatenating all the IDs that would be passed into a single variable but then we aren't even using `passedId`.
Can you argue for the need of this in the resolver?